dotnet/samples/KernelSyntaxExamples/Example03_Variables.cs

```diff
@@ -25,8 +25,20 @@ public static async Task RunAsync()
             ["day"] = DateTimeOffset.Now.ToString("dddd", CultureInfo.CurrentCulture)
         };
 
-        var result = await kernel.InvokeAsync(textPlugin["AppendDay"], arguments);
+        // ** Different ways of executing function with arguments **
 
-        Console.WriteLine(result.GetValue<string>());
+        // Specify and get the value type as generic parameter
+        var resultValue = await kernel.InvokeAsync<string>(textPlugin["AppendDay"], arguments);
+        Console.WriteLine($"string -> {resultValue}");
+
+        // If you need to access the result metadata, you can use the non-generic version to get the FunctionResult
+        var functionResult = await kernel.InvokeAsync(textPlugin["AppendDay"], arguments);
+        var metadata = functionResult.Metadata;
```

Comment:
Do we need this?

Reply:
Added just to justify the `functionResult` usage according to the comment above.

I added all different ways you can retrieve the result, from common to most unlikely. We can remove if this is too much.